1-the-basics/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Raul Valencia</title>
+</head>
+<body>
+    <h1 class="header">Raúl Valencia</h1>
+    <h2>Raúl Valencia</h2>
+    <h3>Raúl Valencia</h3>
```

Comment:
Nice job, you can remove the lines that are repeated. Or you can change the info.

Reply:
Ok, thanks